Doc/library/reprlib.rst

```diff
@@ -123,6 +123,62 @@ which format specific object types.
    similar manner as :attr:`maxstring`.  The default is ``20``.
 
 
+.. attribute:: Repr.indent
+
+   If this attribute is set to ``None`` (the default), the output is formatted
+   with no line breaks or indentation, like the standard :func:`repr`.
+   For example:
+
+   .. code-block:: pycon
+
+      >>> example = [
+              1, 'spam', {'a': 2, 'b': 'spam eggs', 'c': {3: 4.5, 6: []}}, 'ham']
+      >>> print(reprlib.repr(example))
+      [1, 'spam', {'a': 2, 'b': 'spam eggs', 'c': {3: 4.5, 6: []}}, 'ham']
+
+   If :attr:`~Repr.indent` is set to a string, each recursion level
+   is placed on its own line, indented by that string:
+
+   .. code-block:: pycon
+
+      >>> reprlib.aRepr.indent = '-->'
+      >>> print(reprlib.repr(example))
+      [
+      -->1,
+      -->'spam',
+      -->{
+      -->-->'a': 2,
+      -->-->'b': 'spam eggs',
+      -->-->'c': {
+      -->-->-->3: 4.5,
+      -->-->-->6: [],
+      -->-->},
+      -->},
+      -->'ham',
+      ]
+
+   Setting :attr:`~Repr.indent` to a positive integer value behaves as if it
+   was set to a string with that number of spaces:
+
+   .. code-block:: pycon
+
+      >>> reprlib.aRepr.indent = 4
```

Comment:
The documentation doesn't currently recommend monkey-patching attributes on globally used objects. We shouldn't recommend that here either. The danger here is that it's a global state change, which might affect other parts of your process. Like, some third-party library you're using might be using reprlib for a purpose where an indent change (especially non-whitespace indentation) might break it.

Instead, I'd create a new `Repr()` instance in this example and modify `indent` of that object.

Reply:
Sounds good. I have adjusted the docs examples to use a fresh instance of `reprlib.Repr` instead of modifying the global `reprlib.aRepr` object in df43a670847aff53a25c94639f873c28d59cc702. I chose the variable name `aRepr` for the "example instance" to stay consistent with the other example code further down on the `reprlib` docs page.